tests/pytest.ini

```diff
@@ -16,6 +16,7 @@ omit =
     /src/main/urls.py
     /src/manage.py
     */apps.py
+    /src/peoplemeasurement/management/commands/refresh_*
```

Comment:
:thinking: Dit ziet er uit als een pleister op iets dat we eigenlijk al hadden moeten fixen, maar nu issues oplevert omdat de coverage anders onder de 80% valt. Ik zou na deze PR wel graag bovenstaande fixen.

Reply:
Je hebt helemaal gelijk.